sim/common/sod/item_effects/phase_4.go

```diff
@@ -21,6 +21,7 @@ const (
 	TreantsBane              = 228486
 	FistOfTheFireswornMolten = 229374
 	StuddedTimbermawBrawlers = 227809
+	Jeklik                   = 19918
```

Comment:
Could you order this by ID in the list and also use the full weapon name `JekliksCrusher`?

Reply:
Actually I just realized this is a ZG item, so we should create a phase 5 file and put it there. There also seems to be 2 versions (regular and Bloodied) so we'll need to apply the effect to both IDs similar to molten weapons

Reply:
https://www.wowhead.com/classic/item=230911/jekliks-crusher
https://www.wowhead.com/classic/item=231861/jekliks-crusher

Reply:
I'll remove this  for a future PR